src/plugins/discover/public/application/main/components/sidebar/discover_sidebar.tsx

```diff
@@ -325,6 +220,102 @@ export function DiscoverSidebarComponent({
+    useCallback((groupName) => {
+      if (groupName === FieldsGroupNames.AvailableFields) {
+        return {
+          helpText: i18n.translate('discover.fieldChooser.availableFieldsTooltip', {
+            defaultMessage:
+              'Your query returned values for these fields. Click + to add an available field to the data table.',
+          }),
+        };
+      }
+      if (groupName === FieldsGroupNames.PopularFields) {
+        return {
+          helpText: i18n.translate('discover.fieldChooser.popularFieldsTooltip', {
+            defaultMessage: 'Add fields to the table that your organization frequently uses.',
+          }),
+        };
+      }
+      if (groupName === FieldsGroupNames.UnmappedFields) {
+        return {
+          helpText: i18n.translate('discover.fieldChooser.unmappedFieldsTooltip', {
+            defaultMessage:
+              "Add fields to the table that aren't explicitly mapped to a field data type.",
+          }),
+        };
+      }
+    }, []);
+  const fieldsExistenceReader = useExistingFieldsReader();
+  const { fieldGroups, scrollToTopResetCounter } = useGroupedFields({
+    dataViewId: (!isPlainRecord && selectedDataView?.id) || null,
+    fieldsExistenceReader: !isPlainRecord ? fieldsExistenceReader : undefined,
+    allFields: allFields || EMPTY_FIELD_LIST,
+    popularFieldsLimit: !isPlainRecord ? popularFieldsLimit : 0,
+    sortedSelectedFields: selectedFields,
+    isAffectedByGlobalFilter: isGlobalFilterApplied,
+    services: {
+      dataViews,
+    },
+    onFilterField,
+    onSupportedFieldFilter,
+    onOverrideFieldGroupDetails,
+  });
+
+  const renderFieldItem: FieldListGroupedProps<DataViewField>['renderFieldItem'] = useCallback(
+    ({ field, groupName }) => (
+      <li key={`field${field.name}`} data-attr-field={field.name}>
+        <DiscoverField
+          alwaysShowActionButton={alwaysShowActionButtons}
+          field={field}
+          highlight={fieldFilter.name}
+          dataView={selectedDataView!}
+          onAddField={onAddField}
+          onRemoveField={onRemoveField}
+          onAddFilter={onAddFilter}
+          documents$={documents$}
+          trackUiMetric={trackUiMetric}
+          multiFields={multiFieldsMap?.get(field.name)} // ideally we better calculate multifields when they are requested first from the popover
+          onEditField={editField}
+          onDeleteField={deleteField}
+          showFieldStats={showFieldStats}
+          contextualFields={columns}
+          selected={groupName === FieldsGroupNames.SelectedFields || selectedFields.includes(field)}
+        />
+      </li>
+    ),
+    [
+      alwaysShowActionButtons,
+      selectedDataView,
+      onAddField,
+      onRemoveField,
+      onAddFilter,
+      documents$,
+      trackUiMetric,
+      multiFieldsMap,
+      editField,
+      deleteField,
+      showFieldStats,
+      columns,
+      selectedFields,
+      fieldFilter.name,
+    ]
```

Comment:
also a potential follow up cleanup, reduce dependencies here, by creating a separate component, memorized with React.memo ... or extending `DiscoverField` a bit so not wrapper `li` is needed ... nothing for now